exec/plugin-commands-rebuild/src/implementation/index.ts

```diff
@@ -282,6 +282,17 @@ async function _rebuild (
   const warn = (message: string) => {
     logger.info({ message, prefix: opts.dir })
   }
+
+  const allowBuild = (pkgName: string) => {
```

Comment:
there is already an allowBuild function build in `@pnpm/core` called `createAllowBuildFunction`.

Reply:
yes and no.

Yes. There is a private `createAllowBuild` function  https://github.com/pnpm/pnpm/blob/b0afd78339af0daf171474182e02de0096daf529/pkg-manager/core/src/install/index.ts#L1367

But no, it's not exported and don't think it should be exported. Currently only api functions in `@pnpm/core` get exported. But `createAllowBuild` isn't part of it. It's a helper function for installation. If we export it as well, it'd be the only helper function get exported along those apis. I think only exports api functions and create a simple one here is better.

https://github.com/pnpm/pnpm/blob/b0afd78339af0daf171474182e02de0096daf529/pkg-manager/core/src/index.ts#L10

Reply:
Make a new package for it then.

Reply:
what package would you suggest? `@pnpm/utility`?

Reply:
I did the change.